docs/4____network_abstraction.adoc

```diff
@@ -281,27 +281,23 @@ The following table lists the names of MIME types used for the Tx/Rx data variab
 |MIME type
 |Description
 
-|org.fmi-standard.fmi-ls-bus.v1.can
-|Binary variables simulating the automotive CAN network including CAN, CAN FD and CAN XL
+|org.fmi-standard.fmi-ls-bus.can; version="1.0.0-beta.1"
```

Comment:
We should add the media type here, e.g., "application":
application/org.fmi-standard.fmi-ls-bus.can; version="1.0.0-beta.1"

Reply:
> We should add the media type here, e.g., "application": application/org.fmi-standard.fmi-ls-bus.can; version="1.0.0-beta.1"

Yes we can do it if everybody is fine with it. Maybe we can have a short discussion this afternoon, then I can fix it and merge that PR?